src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+import { ThemeProvider } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+
+import theme from "@theme/theme";
+
+import Layout from "@components/Layout/Layout";
```

Comment:
you can add `index.js` file for every component and make default import there
`import { default }  from './Layout'`

It makes imports shorter and that file create possibility to export smth other from this component. Instead import Layout from `"@components/Layout/Layout";` it will be import Layout from `"@components/Layout"`